source/Halibut/Transport/SecureListener.cs

```diff
@@ -113,7 +113,7 @@ async Task Accept()
                     {
                         log.WriteException(EventType.Error, "Error accepting TCP client", ex);
                         // Slow down the logs in case an exception is immediately encountered each AcceptTcpClientAsync
-                        Thread.Sleep(1000);
+                        await Task.Delay(1000);
```

Comment:
Changing this would make it hard to see this loop in a debug dump

Reply:
I'm not super tied to this change. Potentially a thread sleep could cause a deadlock because it hogs a thread that could be returned to the async pool, but in practice I doubt that would happen anyway.

Reply:
Yeh I think we should keep it the way it is, it's super useful when debugging